week06/Convert_Sorted_Array_to_Binary_Search_Tree.cpp

```diff
@@ -0,0 +1,17 @@
+/// TC: O(NlogN) /// N = Size of nums array
+/// MC: O(T) T means per recursion call
```

Comment:
It is vague.

It should be `O(H)` where H is the height of the tree.

Reply:
ok, bro...